2025-09-15 02:09:05,698 INFO    MainThread:13483 [wandb_init.py:setup_run_log_directory():686] Logging user logs to /workspace/wandb/run-20250915_020905-kwemu4gv/logs/debug.log
2025-09-15 02:09:05,701 INFO    MainThread:13483 [wandb_init.py:setup_run_log_directory():687] Logging internal logs to /workspace/wandb/run-20250915_020905-kwemu4gv/logs/debug-internal.log
2025-09-15 02:09:05,703 INFO    MainThread:13483 [wandb_init.py:monkeypatch_ipython():605] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x72a572e6bc50>
2025-09-15 02:09:05,707 INFO    MainThread:13483 [wandb_init.py:init():813] calling init triggers
2025-09-15 02:09:05,708 INFO    MainThread:13483 [wandb_init.py:init():818] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.005, 'epochs': 20, 'batch_size': 128, 'optimizer': 'adam', '_wandb': {}}
2025-09-15 02:09:06,665 INFO    MainThread:13483 [wandb_init.py:init():854] starting backend
2025-09-15 02:09:06,666 INFO    MainThread:13483 [wandb_init.py:init():857] sending inform_init request
2025-09-15 02:09:06,716 INFO    MainThread:13483 [wandb_init.py:init():865] backend started and connected
2025-09-15 02:09:07,100 INFO    MainThread:13483 [wandb_run.py:_label_probe_notebook():1330] probe notebook
2025-09-15 02:09:07,130 INFO    MainThread:13483 [wandb_init.py:init():936] updated telemetry
2025-09-15 02:09:07,135 INFO    MainThread:13483 [wandb_init.py:init():960] communicating run to backend with 90.0 second timeout
2025-09-15 02:09:07,419 INFO    MainThread:13483 [wandb_init.py:init():1011] starting run threads in backend
2025-09-15 02:09:08,062 INFO    MainThread:13483 [wandb_run.py:_console_start():2506] atexit reg
2025-09-15 02:09:08,064 INFO    MainThread:13483 [wandb_run.py:_redirect():2354] redirect: wrap_raw
2025-09-15 02:09:08,065 INFO    MainThread:13483 [wandb_run.py:_redirect():2423] Wrapping output streams.
2025-09-15 02:09:08,068 INFO    MainThread:13483 [wandb_run.py:_redirect():2446] Redirects installed.
2025-09-15 02:09:08,076 INFO    MainThread:13483 [wandb_init.py:init():1049] run started, returning control to user process
2025-09-15 02:09:08,087 INFO    MainThread:13483 [wandb_init.py:_post_run_cell_hook():575] [no run ID] resuming backend
2025-09-15 02:09:23,011 INFO    MainThread:13483 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-09-15 02:09:23,012 INFO    MainThread:13483 [wandb_init.py:_pre_run_cell_hook():564] [no run ID] pausing backend
2025-09-15 02:09:23,749 INFO    MainThread:13483 [wandb_init.py:_post_run_cell_hook():575] [no run ID] resuming backend
2025-09-15 02:09:25,958 INFO    MainThread:13483 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-09-15 02:09:25,959 INFO    MainThread:13483 [wandb_init.py:_pre_run_cell_hook():564] [no run ID] pausing backend
2025-09-15 02:09:26,017 INFO    MainThread:13483 [wandb_init.py:_post_run_cell_hook():575] [no run ID] resuming backend
2025-09-15 02:09:27,604 INFO    MainThread:13483 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-09-15 02:09:27,605 INFO    MainThread:13483 [wandb_init.py:_pre_run_cell_hook():564] [no run ID] pausing backend
2025-09-15 02:11:20,025 INFO    MainThread:13483 [wandb_init.py:_post_run_cell_hook():575] [no run ID] resuming backend
2025-09-15 02:10:33,310 INFO    MainThread:13483 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-09-15 02:10:33,312 INFO    MainThread:13483 [wandb_init.py:_pre_run_cell_hook():564] [no run ID] pausing backend
2025-09-15 02:10:33,957 INFO    MainThread:13483 [wandb_init.py:_post_run_cell_hook():575] [no run ID] resuming backend
2025-09-15 02:13:47,568 INFO    MainThread:13483 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-09-15 02:13:47,577 INFO    MainThread:13483 [wandb_init.py:_pre_run_cell_hook():564] [no run ID] pausing backend
2025-09-15 02:13:47,945 INFO    MainThread:13483 [wandb_run.py:_finish():2272] finishing run emilio-/Reconocimiento MNIST/kwemu4gv
2025-09-15 02:13:47,960 INFO    MainThread:13483 [jupyter.py:save_history():480] not saving jupyter history
2025-09-15 02:13:47,961 INFO    MainThread:13483 [jupyter.py:save_ipynb():404] not saving jupyter notebook
2025-09-15 02:13:47,961 INFO    MainThread:13483 [wandb_init.py:_jupyter_teardown():590] cleaning up jupyter logic
2025-09-15 02:13:47,966 INFO    MainThread:13483 [wandb_run.py:_atexit_cleanup():2471] got exitcode: 0
2025-09-15 02:13:47,975 INFO    MainThread:13483 [wandb_run.py:_restore():2453] restore
2025-09-15 02:13:47,981 INFO    MainThread:13483 [wandb_run.py:_restore():2459] restore done
2025-09-15 02:13:48,880 INFO    MainThread:13483 [wandb_run.py:_footer_sync_info():3867] logging synced files
